packages/atlas-react/rollup.config.js

```diff
@@ -18,13 +20,16 @@ export default [
       scss({
         prefix: `@import "./src/common/styles/atlas.scss";`,
         output: "dist/atlas.css",
+        outputStyle:
+          process.env.NODE_ENV === "production" ? "compressed" : "expanded",
       }),
       commonjs({ include: /node_modules/ }),
-      babel({ babelHelpers: 'bundled' })
+      babel({ babelHelpers: "bundled" }),
+      process.env.NODE_ENV === "production" && uglify(),
     ],
     output: [
-      { dir: './dist/cjs/', format: "cjs" },
-      { dir: './dist/esm/', format: "es", exports: "named" },
+      { dir: "./dist/cjs/", format: "cjs", sourcemap: true },
+      { dir: "./dist/esm/", format: "es", exports: "named", sourcemap: true },
```

Comment:
adding the sourcemaps, noticed that it's somehow hard to `debug` on chrome after deployed/distributed.